contracts/contracts/sbtc-registry.clar

```diff
@@ -234,7 +236,9 @@
       topic: "completed-deposit",
       bitcoin-txid: txid,
       output-index: vout-index,
-      amount: amount
+      amount: amount,
+      burn-hash: burn-hash,
+      burn-height: burn-height,
```

Comment:
We should add these same fields for the print event for withdrawals too.

Reply:
Done